english/destiny/destiny.md

```diff
@@ -22,14 +20,14 @@ and you might be someone [who knocks](../say-my-name/say-my-name.md).
 
 or perhaps you are someone [who opens](https://www.biblegateway.com/passage/?search=Revelation+3%3A20).
 
-If you have finished pursuing your destiny, it may be time for a 
+If you have finished pursuing your destiny, it may be time for a
 [cold one](../drink-beer/drink-beer.md).
 
 If you have only just begun to pursue your destiny, it's best not to
-do so on an [empty stomach](../breakfast/breakfast.md). 
+do so on an [empty stomach](../breakfast/breakfast.md).
+
+You stop to contemplate. Deep within your mind you can't help but question whether your mind and body are separable and if this is truly [reality](../reality/reality.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.